prime-router/docs/onboarding-users/transport/rest.md

```diff
@@ -0,0 +1,160 @@
+# REST Transport
+## Introduction
+ReportStream supports four popular transport protocols:  Simple Object Access Protocol (SOAP), Resource State Transfer (REST), Secure File Transfer Protocol (SFTP), and Azure Blob. The goal of this document is to explain how to configure REST Transport for STLTs (a/k/a receivers). As well as provide guidance on how: 1) ReportStream REST Transport works, 2) to prepare a key for authentication, 3) to configure receiver's REST transport, and 4) test/check that a connection is successful. Please see ABC document on how to configure other transports.
```

Comment:
ABC document?

Reply:
That's a placeholder for future documentation on the other transports. Possibly remove that line?

Reply:
There's already existing documentation that it refers to on other transport types...

Reply:
Is there a link we can add?

Reply:
I would suggest just link the  relevant doc to the stated transport protocol in the first sentence (e.g just have "Simple Object Access Protocol (SOAP) be a link to the .md file).

Reply:
```suggestion
ReportStream supports four popular transport protocols:  Resource State Transfer (REST), [Simple Object Access Protocol (SOAP)](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/onboarding-users/transport/soap.md), [Secure File Transfer Protocol (SFTP)](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/onboarding-users/transport/sftp.md), and [Azure Blob](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/onboarding-users/transport/blob.md). The goal of this document is to explain how to configure REST Transport for STLTs (a/k/a receivers). As well as provide guidance on how: 1) ReportStream REST Transport works, 2) to prepare a key for authentication, 3) to configure receiver's REST transport, and 4) test/check that a connection is successful. Please see ABC document on how to configure other transports.
```

Reply:
Moved REST to the beginning, too, since it's this document and thus won't have a link.

Reply:
Thank you for all of the comments.

Reply:
Fixed.